spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -12,6 +12,33 @@ module RSpec::Rails
       end
     end
 
+    context "with use_transactional_tests set to true" do
+      it "works with #uses_transaction helper" do
+        group = RSpec::Core::ExampleGroup.describe do
+          include FixtureSupport
+          self.use_transactional_tests = true
+
+          uses_transaction "doesn't run in transaction"
+
+          it "doesn't run in transaction" do
+            expect(ActiveRecord::Base.connection.transaction_open?).to eq(false)
```

Comment:
I'm having a big WTF moment at this, so `uses_transaction` turns off transactional tests?

Reply:
Yes. No doubt `uses_transaction` is an ambiguous and confusing naming.